server/license.go

```diff
@@ -0,0 +1,50 @@
+package server
+
+import (
+	"net/http"
+
+	enterpriseAPI "github.com/bytebase/bytebase/enterprise/api"
+	"github.com/google/jsonapi"
+	"github.com/labstack/echo/v4"
+)
+
+func (s *Server) registerLicenseRoutes(g *echo.Group) {
+	g.GET("/license", func(c echo.Context) error {
```

Comment:
I'm thinking about using `GET /subscription` (return features, including plan type, instance count and expires time) and `PATCH /subscription` (update license token) to combine the `/plan` and `/license` endpoints. Any suggestions

Reply:
SGTM